code/datums/diseases/advance/symptoms/neurological.dm

```diff
@@ -0,0 +1,77 @@
+/datum/symptom/nerve_overdrive
+	name = "Neurological Overdrive"
+	desc = "The disease increases connectivity between the infected's nerves, reducing stuns."
+	stealth = 1
+	resistance = -1
+	stage_speed = 1
+	transmittable = -2
+	level = 9
+	severity = 0
+	symptom_delay_min = 1
+	symptom_delay_max = 1
+	var/fastmover = FALSE
+	var/fastmover_activation = 0 //Each of the Indexes Represents a Stage, When Activated, the Index is set to true
+	var/super_reduction = FALSE
+	threshold_desc = "<b>Stage Speed 12:</b>The virus overclocks the nervous system making the host move even faster than normal.<br>\
+                      <b>Stage Speed 15:</b>The virus tightens the synapses between nerves, allowing for even quicker recovery from stuns."
+
+/datum/symptom/nerve_overdrive/OnAdd(datum/disease/advance/A)
+	A.infectable_biotypes |= MOB_ROBOTIC //Robots don't have nerves.
```

Comment:
Manipulating `infectable_biotypes` should be kept at minimum. Most diseases don't have `MOB_ROBOTIC` in the first place, too.

Reply:
If robots dont have nerves why are you making robots susceptable?

Reply:
Wait I thought that removed it.